src/components/Popup.js

```diff
@@ -9,6 +9,14 @@ export default class Popup {
     }
   };
 
+  loading(yes) {
+    if (yes) {
+      this._submitBtn.textContent = "Saving...";
+    } else {
+      this._submitBtn.textContent = "Save";
+    }
```

Comment:
Please, delete `loading` from `Popup` because it's not needed for the image modal

There should **not** be any code in universal functions that doesn’t suit every popup.